common/presenters/index.js

```diff
@@ -22,6 +22,7 @@ const moveTypesToFilterComponent = require('./move-type-for-filter')
 const movesByLocation = require('./moves-by-location')
 const movesToCSV = require('./moves-to-csv')
 const personToSummaryListComponent = require('./person-to-summary-list-component')
+const locationsToPopulationTableComponent = require('./populationTable/locations-to-population-table-component')
```

Comment:
Can we use kebabcase for the directory please? Be good to stay consistent with the rest of the codebase :)

Reply:
Done :)